### PR TITLE
New version: Jackett.Jackett version 0.20.1001.0 [FP]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Jackett | ImageMagick 7.1.0-32 Q16-HDRI (64-bit) (2022-04-30), DBeaver 22.0.4 (current user), Google Chrome Canary    |
| Version     | 0.20.1001.0     | 7.1.0.32, 22.0.4, 103.0.5038.0 |
| Publisher   | Jackett   | ImageMagick Studio LLC, DBeaver Corp, Google LLC      |
| ProductCode |           | ImageMagick 7.1.0 Q16-HDRI (64-bit)_is1, DBeaver (current user), Google Chrome SxS    |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [1363](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/2258154525>)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Version Number incorrect

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.installer.yaml [hunk 1]
```diff
@@ -2,20 +2,20 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1934
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.5-aptest.1934
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1934
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.5-aptest.1934
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.5-alpha.1923
+PackageVersion: 1.4.5.1934
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.5.1934/CiderCollective.Cider.Nightly.yaml:5]:
```suggestion
PackageVersion: 1.4.5-aptest.1934
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**